scania-digital-design-system/tegel
# fix(button): colors update

**Describe pull-request**  
Buttons color update

**Solving issue**  
The button had some hard coded values for variations of black and white color with different opacity.
New color tokens are added as they are used in multiple places. 

Fixes: [CDEP-2999](https://tegel.atlassian.net/browse/CDEP-2999)

**How to test**  
1. Go to button component
2. Check colors for ghost, secondary and danger button in both dark and light mode
3. Check disabled, hover and active (pressed) state
4. Compare it with Figma
5. Color values need to match

**Suggested test steps**



[CDEP-2999]: https://tegel.atlassian.net/browse/CDEP-2999?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
